images/runtime/dotnetcore/8.0/base.bookworm.Dockerfile

```diff
@@ -0,0 +1,62 @@
+# dotnet tools are currently available as part of SDK so we need to create them in an sdk image
+# and copy them to our final runtime image
+FROM mcr.microsoft.com/dotnet/sdk:8.0-preview AS tools-install
+RUN dotnet tool install --tool-path /dotnetcore-tools dotnet-sos
+RUN dotnet tool install --tool-path /dotnetcore-tools dotnet-trace
+RUN dotnet tool install --tool-path /dotnetcore-tools dotnet-dump
+RUN dotnet tool install --tool-path /dotnetcore-tools dotnet-counters
+RUN dotnet tool install --tool-path /dotnetcore-tools dotnet-gcdump
+RUN dotnet tool install --tool-path /dotnetcore-tools dotnet-monitor --version 8.0.0-preview.7.23402.4
```

Comment:
Note that for the final version, you'll want dotnet-monitor --version 8.* to make sure that the latest minor version is getting picked up in the oryx image. 

For previews, we do update monthly alongside .net 8 previews, so this would need to be updated as part of the sdk updates.